### PR TITLE
Firebase-Authentication guide added to backend/authentication.json

### PR DESCRIPTION
**Pull request by SoumyadeepOSD**
## Screenshots
image

### CODE REVIEW

**Comment by rupali-codes** [comment]
@SoumyadeepOSD are you a gssoc contributor?

**Comment by SoumyadeepOSD** [comment]
> @SoumyadeepOSD are you a gssoc contributor?

yes

File: database/backend/authentication.json [hunk 1]
```diff
@@ -1 +1,9 @@
-[]
+[
+    {
+        "name": "Firebase Authentication",
+        "description": "This guide provides best practices and patterns for building applications ny using Firebase.",
```
Comment (CBID2) [inline database/backend/authentication.json:4]:
```suggestion
        "description": "This guide provides information on best practices and patterns for building applications using Firebase.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hey @SoumyadeepOS! You’re off to a great start! :) Just refer to my inline comment and you're good to go! :)

**Review by CBID2 — APPROVED** [review]
Looks good @SoumyadeepOSD! :) LGTM! :)

**Comment by CBID2** [comment]
@rupali-codes, we need a force push here.

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**